src/guide/typescript-support.md

```diff
@@ -133,6 +133,47 @@ const Component = defineComponent({
 })
 ```
 
+### Augmenting Types for Use with Plugins or User Definitions
```

Comment:
I don't use TypeScript myself so this might be a really dumb question, but is 'User Definitions' a technical term? I don't really understand what it means here (or where it's used in the example).

Reply:
My English is so poor that I can't describe my thoughts accurately. But I read the relevant documentation of TypeScript to learn the exact definition, and found that they use "declaration" to describe a type, so I think it is better to replace it with "declarations by user".